src/plays/motivate-me/MotivateMe.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import { quotes as Quotes } from './data';
+import Header from './components/header';
+import './index.css';
+import Container from './container';
+import Footer from './components/footer';
+function GetMotivated() {
+	const [author, setAuthor] = useState('');
+	return (
+		<div className="main">
+			<Header />
+			<Container author={author} setAuthor={setAuthor} Quotes={Quotes} />
+			<Footer />
+		</div>
+	);
+}
+
+export default GetMotivated;
```

Comment:
@Hat52, We should not directly pass the `setAuthor` method as a prop.

Create a handler and set the author value there, then pass that handler to as a prop.

```js
const handleAuthorChange=(value)=> setAuthor(value)


```